core/src/main/java/fj/Equal.java

```diff
@@ -292,6 +292,11 @@ public boolean equal(A a1, A a2) {
   public static final Equal<BitSet> bitSetSequal = equalDef((bs1, bs2) -> bs1.longValue() == bs2.longValue());
 
   /**
+   * An equal instance for the {@link Stream} of {@link Character}.
+   */
+  public static final Equal<Stream<Character>> streamCharacterEqual = streamEqual(charEqual);
```

Comment:
I think this is a bit too much of specific to be part of the public api.
Can we put in a static private nested class of LazyString?